tests/run/cpp_stl_random.pyx

```diff
@@ -53,3 +62,302 @@ def mt19937_discard(z):
     for _ in range(z + 1):
         b = rd()
     return a, b
+
+
+ctypedef fused any_dist:
```

Comment:
That's a nice trick to force it to compile the tests for all the combinations to distributions.